frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/PipelineRunDetails.tsx

```diff
@@ -1,7 +1,9 @@
 import * as React from 'react';
-import { SectionHeading, ResourceSummary } from '@console/internal/components/utils';
+import { SectionHeading, ResourceSummary, ResourceLink } from '@console/internal/components/utils';
 import PipelineRunVisualization from './PipelineRunVisualization';
 import { PipelineRun } from '../../../utils/pipeline-augment';
+import { referenceForModel } from '@console/internal/module/k8s';
```

Comment:
Can you sort this to the top of the imports? Also, is there a more simple import like just `@console/internal`? (not sure, just wondering)

Reply:
I searched but couldn't find any import like `@console/internal`